.github/workflows/codegen.yml

```diff
@@ -8,13 +8,15 @@ jobs:
   codegen:
     runs-on: ubuntu-latest
     steps:
-    - name: checkout
-      uses: actions/checkout@v1
-
     - uses: actions/setup-go@v2
       with:
         go-version: 1.15
 
+    - name: checkout
+      uses: actions/checkout@v2
+      with:
+        path: $(go env GOPATH)/src
+
     - name: run codegen
       run: GOPATH=$(go env GOPATH) make codegen
```

Comment:
I think we need to change dir; see `working-directory`

Reply:
`Working directory is '/home/runner/work/rook/rook/$(go env GOPATH)/src'`... I change manually to point `"/home/runner/go/src"` but no luck do you see what I'm missing. @leseb

Reply:
ahh. it worked...now I see. Thanks @leseb